packages/opentelemetry-core/test/metrics/NoopMeter.test.ts

```diff
@@ -0,0 +1,71 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+import { DistributedContext, SpanContext } from '@opentelemetry/types';
+import {
+  NoopMeter,
+  NOOP_GAUGE_HANDLE,
+  NOOP_GAUGE_METRIC,
+  NOOP_COUNTER_HANDLE,
+  NOOP_COUNTER_METRIC,
+  NOOP_MEASURE_HANDLE,
+  NOOP_MEASURE_METRIC,
+} from '../../src/metrics/NoopMeter';
+
+describe('NoopTracer', () => {
+  it('should not crash', () => {
+    const meter = new NoopMeter();
+    const counter = meter.createCounter('some-name');
```

Comment:
Could you please add tests for `createCounter`  with `MetricOptions`, same with `Gauge` and `Measure`.